### PR TITLE
make environment variables the sole source of configuration DEV-696

### PR DESCRIPTION
**Pull request by vbrik**

### CODE REVIEW

File: .ripgreprc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+--hidden
+--glob=!.git/*
```
Comment (blinkdog) [inline .ripgreprc]: Echoing David's comment, I wasn't sure about this file, but found this:
https://github.com/BurntSushi/ripgrep
Interesting tool, I haven't heard of this one before.

**Comment by vbrik** [comment]
> Looks good to me. What is the `.ripgreprc` file?

ripgrep is a more convenient alternative to grep (e.g. it automatically ignores files from .gitignore). Without `--hidden` it wouldn't pick up .circleci/config.yml when I was grepping for environment variables

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,6 +18,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Security
 - in case of vulnerabilities
 
+## [1.1.1] - 2019-08-13
```
Comment (blinkdog) [inline CHANGELOG.md]: Need to update the link(s) at the bottom of the file so this [1.1.1] points to something.
I was going to propose a quick edit, but GitHub is fighting with me about being on a branch in order to propose changes, blah blah

File: README.md [hunk 3]
```diff
@@ -21,8 +21,10 @@ Then run it with:
     circleci local execute --job test
 
 ## Configuration
-By default, the service listens on port 8888. This is specified
-in `server.py` in the constructor for the `Server` class.
+All configuration is done using environment variables.
+To get the list of possible configuration parameters and their defaults, run
+
+    python -m file_catalog --show-config-spec
```
Comment (blinkdog) [inline README.md]: This is appreciated. Being able to interrogate the program as to what it thinks its configuration is seems very useful for debugging.

**Pull request merged**